test/utils/browser.js

```diff
@@ -81,6 +144,11 @@ exports.assertBrowser = function () {
   });
 };
 
+exports.simpleExecute = function (fn, args, cb) {
+  // DEV: Barebones execute function on the browser, this works when .execute does not
+  this.browser.execute(functionToString(fn).body, args, cb);
+};
```

Comment:
Checked there, and the selenium driver output.  Nothing... :-1: 

I'll have another crack at it tonight